### PR TITLE
github pull request properties

### PR DESCRIPTION
**Pull request by p3ck**

### CODE REVIEW

File: config/Dockerfiles/JenkinsfileStageMerge [hunk 1]
```diff
@@ -16,6 +16,33 @@ library identifier: "ci-pipeline@master",
...
 ])
 
+properties([
+  buildDiscarder(logRotator(artifactNumToKeepStr: '20', numToKeepStr: '20')),
+  [$class: 'GithubProjectProperty', displayName: '', projectUrlStr: 'https://github.com/CentOS-PaaS-SIG/linchpin/'],
+  [$class: 'org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty', triggers:[
+    [
+      $class: 'org.jenkinsci.plugins.ghprb.GhprbTrigger',
+      orgslist: 'CentOS-PaaS-SIG',
+      cron: 'H/5 * * * *',
```
Comment (arilivigni) [inline config/Dockerfiles/JenkinsfileStageMerge]: Why are you also using CRON?

File: config/Dockerfiles/JenkinsfileStageTrigger [hunk 2]
```diff
@@ -36,6 +36,33 @@ library identifier: "cico-pipeline-library@master",
...
                               remote: "https://github.com/CentOS/cico-pipeline-library"])
 
+properties([
+  buildDiscarder(logRotator(artifactNumToKeepStr: '20', numToKeepStr: '20')),
+  [$class: 'GithubProjectProperty', displayName: '', projectUrlStr: 'https://github.com/CentOS-PaaS-SIG/linchpin/'],
+  [$class: 'org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty', triggers:[
+    [
+      $class: 'org.jenkinsci.plugins.ghprb.GhprbTrigger',
+      orgslist: 'CentOS-PaaS-SIG',
+      cron: 'H/5 * * * *',
```
Comment (arilivigni) [inline config/Dockerfiles/JenkinsfileStageTrigger]: Why are you also using CRON?

Comment (arilivigni) [reply config/Dockerfiles/JenkinsfileStageTrigger]: Shouldn't you only need the keyword only?

File: config/Dockerfiles/JenkinsfileStageMerge (hunk 1 above)
Comment (arilivigni) [reply config/Dockerfiles/JenkinsfileStageMerge]: Shouldn't you only need the keyword only?

File: config/Dockerfiles/JenkinsfileStageTrigger [hunk 3]
```diff
@@ -36,6 +36,33 @@ library identifier: "cico-pipeline-library@master",
...
+      onlyTriggerPhrase: false,
+      useGitHubHooks: true,
+      permitAll: true,
+      autoCloseFailedPullRequests: false,
+      displayBuildErrorsOnDownstreamBuilds: true,
+      extensions: [
+        [
+          $class: 'org.jenkinsci.plugins.ghprb.extensions.status.GhprbSimpleStatus',
+          commitStatusContext: 'Docker Build',
+          showMatrixStatus: false,
```
Comment (arilivigni) [inline config/Dockerfiles/JenkinsfileStageTrigger]: Why include props if they are already defaulted to false or true?

Comment (p3ck) [reply config/Dockerfiles/JenkinsfileStageTrigger]: Do we know what the defaults are?

Comment (arilivigni) [reply config/Dockerfiles/JenkinsfileStageTrigger]: in the pipeline syntax I believe if you go in blank it should show you

File: config/Dockerfiles/JenkinsfileStageMerge (hunk 1 above)
Comment (arilivigni) [reply config/Dockerfiles/JenkinsfileStageMerge]: ok

Comment (p3ck) [reply config/Dockerfiles/JenkinsfileStageMerge]: What do you mean by "only need the keyword only?"

Comment (arilivigni) [reply config/Dockerfiles/JenkinsfileStageMerge]: like [test]  so you are not polling but now your explanation on minishift makes sense

File: config/Dockerfiles/JenkinsfileStageTrigger (hunk 3 above)
Comment (arilivigni) [reply config/Dockerfiles/JenkinsfileStageTrigger]: This is a suggestion shouldn't hold up the change.

**Comment by herlo** [comment]
[test]

**Comment by p3ck** [comment]
[test]

**Comment by herlo** [comment]
[test]

**Comment by herlo** [comment]
[merge]

**Pull request merged**